docs/general-reference/help-button.md

```diff
@@ -0,0 +1,113 @@
+---
+layout: default
+title: Firebolt Help
+description: Describes the options of the Firebolt "Help" button
+nav_exclude: true
```

Comment:
is excluding it from the nav (left menu) intentional?

Reply:
No, fixed